api/turing/api/routers_api.go

```diff
@@ -267,6 +298,11 @@ func (c RoutersController) DeployRouter(
 		return NotFound("router version not found", err.Error())
 	}
 
+	// call webhook for router deployment event
+	if errWebhook := c.webhookClient.TriggerRouterEvent(ctx, webhook.OnRouterDeployed, router); errWebhook != nil {
```

Comment:
I think need to add this event on router creation/update too, because in that endpoint there's also a mechanism to deploy a router, or maybe add this event inside the function of `deployOrRollbackRouter()` 🤔 What do you think?

Reply:
Fixed, thanks @bthari for your detail 🙏